src/libs/EmojiUtils.js

```diff
@@ -89,20 +89,40 @@ function getDynamicHeaderIndices(emojis) {
 /**
  * Get number of empty spaces to be filled to get equal emojis for every row
  * @param {Number} emojiCount
+ * @param {Number} suffix
  * @returns {Object[]}
  */
-function getDynamicSpacing(emojiCount) {
+function getDynamicSpacing(emojiCount, suffix) {
     const spacerEmojis = [];
     let modLength = CONST.EMOJI_NUM_PER_ROW - (emojiCount % CONST.EMOJI_NUM_PER_ROW);
     while (modLength > 0) {
         spacerEmojis.push({
-            code: CONST.EMOJI_SPACER,
+            code: `${CONST.EMOJI_SPACER}_${suffix}_${modLength}`,
+            spacer: true,
         });
         modLength -= 1;
     }
     return spacerEmojis;
 }
 
+/**
+ * Add dynamic spaces to emoji categories
+ * @param {Object[]} emojis
+ * @returns {Object[]}
+ */
+function addSpacesToEmojiCategories(emojis) {
+    let updatedEmojis = [];
+    _.each(emojis, (emoji, index) => {
+        if (emoji.header) {
+            updatedEmojis = updatedEmojis.concat(getDynamicSpacing(updatedEmojis.length, index), [emoji], getDynamicSpacing(1, index));
+        } else {
```

Comment:
return early here

Reply:
I am not sure when you say return early. There's an if and else case to it.  Neither it gave an error for the same.

Reply:
I mean to say - return at end of `if`
So you don't need to use `else`